editor/plugins/script_editor_plugin.cpp

```diff
@@ -1684,7 +1684,19 @@ void ScriptEditor::_update_script_names() {
 			Ref<Texture> icon = se->get_icon();
 			String path = se->get_edited_resource()->get_path();
 			bool built_in = !path.is_resource_file();
-			String name = built_in ? path.get_file() : se->get_name();
+			String name;
+
+			if (built_in) {
+
+				name = path.get_file();
+				String resourceName = se->get_edited_resource()->get_name();
```

Comment:
Please use `snake_case` for identifiers, so this should be `resource_name`.

Reply:
I've updated this to use snake case.